nirc_ehr/resources/queries/study/arrival.js

```diff
@@ -81,7 +81,8 @@ EHR.Server.TriggerManager.registerHandlerForQuery(EHR.Server.TriggerManager.Even
                 date: row.date,
                 cage: row.cage,
                 taskid: row.taskid,
-                qcstate: row.qcstate
+                qcstate: row.qcstate,
+                reason: 'Non-quarantine Arrival'
```

Comment:
We may need to clear this up in our next meeting but I don't think we want this hard coded. I think the arrival form should relabel the housing "Reason" as "Arrival Type", then by default the Non-quarantine Arrival option is selected. But a user can change that if needed.

Reply:
There isn't a housing 'reason' column in Arrival form unless we add it like we do other demographics cols. Also, there is a new 'acquisition type' column, which will get confusing for users if we add "arrival type" and "acquisition type".

Reply:
Well lets talk about it today in the meeting. Somehow we need to cover all the scenarios.

Reply:
Arrival type now added with lookup values